OHHTTPStubs/Sources/OHHTTPStubs.m

```diff
@@ -43,7 +43,7 @@ @interface OHHTTPStubsProtocol : NSURLProtocol @end
 
 @interface OHHTTPStubs()
 + (instancetype)sharedInstance;
-@property(atomic, copy) NSMutableArray* stubDescriptors;
+@property(atomic, readonly) NSMutableArray* stubDescriptors;
```

Comment:
Problem with this is, it's a reference type. So that means that users can't set `stubDescriptors = newValue` but they can `[stubDescriptors append:…]` though can't they?

Reply:
Oh wait it's in the private category so not exposed to the consumer is it? (Not easy to review from phone 😅)

Reply:
(Forgot to reply originally) `copy` never works on properties of `NSMutableCopying` type, because the setter calls `-copy`, which returns an immutable copy, i.e. an `NSArray` in this case. So you’d set a mutable array, and the value that gets stored would be immutable. Fun edge case in Obj-C land that should probably be a compile error.